Makefile

```diff
@@ -136,19 +146,31 @@ help:  ## Display this help
 test: ## Run tests
 	source ./scripts/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v ./... -coverprofile cover.out
 
-# Allow overriding the e2e configurations
+#
+# Ginkgo configuration.
+#
+GINKGO_FOCUS ?=
+GINKGO_SKIP ?=
 GINKGO_NODES ?= 1
+GINKGO_TIMEOUT ?= 6h
+GINKGO_POLL_PROGRESS_AFTER ?= 60m
```

Comment:
I like the addition of progress report polling here...hopefully this will make it easier to see what's going on if this PR happens to _not_ fully fix the e2e run.

Reply:
Me too. I don't know if it does anything, but hey.